# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: toolkit/library/rust/shared/Cargo.toml
# Commit: 96fd1bb41036
# Full Hash: 96fd1bb41036f14d9b18f9d1e462f8382453c6f2
# Author: Matthew Gregan <kinetik@flim.org>
# Date: 2021-09-12 09:05:27
# Description:
#   Bug 1725749 - Update audioipc to fce878ff.  r=chunmin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D125165
# ==============================================================================

diff -r 64212702d9f0 -r 96fd1bb41036 toolkit/library/rust/shared/Cargo.toml
--- a/toolkit/library/rust/shared/Cargo.toml	Sat Sep 11 22:52:58 2021 +0300
+++ b/toolkit/library/rust/shared/Cargo.toml	Sat Sep 11 22:34:31 2021 +0000
@@ -23,8 +23,8 @@
 cubeb-pulse = { git = "https://github.com/mozilla/cubeb-pulse-rs", rev="3ad5978575f501ab10b1753626f176f1bba3f584", optional = true, features=["pulse-dlopen"] }
 cubeb-sys = { version = "0.9", optional = true, features=["gecko-in-tree"] }
 encoding_glue = { path = "../../../../intl/encoding_glue" }
-audioipc-client = { git = "https://github.com/mozilla/audioipc-2", rev = "8bb1a227fbaa5677458bcd876162b65307df38c2", optional = true }
-audioipc-server = { git = "https://github.com/mozilla/audioipc-2", rev = "8bb1a227fbaa5677458bcd876162b65307df38c2", optional = true }
+audioipc-client = { git = "https://github.com/mozilla/audioipc-2", rev = "fce878ff15f4e1dba1e5b54f82460af605fb4d14", optional = true }
+audioipc-server = { git = "https://github.com/mozilla/audioipc-2", rev = "fce878ff15f4e1dba1e5b54f82460af605fb4d14", optional = true }
 authenticator = "0.3.1"
 gkrust_utils = { path = "../../../../xpcom/rust/gkrust_utils" }
 gecko_logger = { path = "../../../../xpcom/rust/gecko_logger" }
